src/ResourceManager/Aks/Commands.Aks/Az.Aks.psd1

```diff
@@ -1,5 +1,5 @@
 #
-# Module manifest for module 'PSGet_AzureRM.Aks'
+# Module manifest for module 'AzureRM.Aks'
```

Comment:
Shouldn't the `Az` psd1 file headers say `Az.Aks`?

Reply:
Yep, good catch. Will update.